app/services/ExposureNotificationService.ts

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line
+declare const global: any;
+
+global.ExposureNotificationsOn = true;
+
+type ENStatusReason =
+  | 'DEVICE_EXPOSURE_NOTIFICATIONS_OFF'
+  | 'ALL_CONDITIONS_MET';
+
+interface ExposureNotificationsState {
+  canTrack: boolean;
+  reason: ENStatusReason;
+  hasPotentialExposure: boolean;
+}
+
+export default class ExposureNotificationService {
```

Comment:
What do you think about not using classes here?

It also looks like a lot of this might want to live in the context component.

Reply:
We used the class to match the implementation details of the Location service. Trying to minimize the isGPS logic as much as possible in these shared components. I can look into refactoring the Main.js a bit to make this work with a context, but then this might cause us to but heads with the GPS implementation.

Reply:
thanks for the context, makes sense to leave as is and we can do any refactorings later.